doc/conf.py

```diff
@@ -351,6 +351,7 @@
 		('boards/arm/nrf52_pca10040/doc/nrf52_pca10040', 'boards/arm/nrf52_pca10040/doc/index'),
 		('boards/arm/nrf52_pca20020/doc/nrf52_pca20020', 'boards/arm/nrf52_pca20020/doc/index'),
 		('boards/arm/nrf52_vbluno52/doc/nrf52_vbluno52', 'boards/arm/nrf52_vbluno52/doc/index'),
+		('boards/arm/nrf5340_dk_nrf5340/doc/nrf5340_dk_nrf5340/index', 'boards/arm/nrf5340pdk_nrf5340/doc/index'),
```

Comment:
I think this redirect is not useful because you are using the old links to the doc and  this board didn't exist before we changed the paths. I think this should be:

```
('boards/arm/nrf5340_dk_nrf5340/doc/index'', 'boards/arm/nrf5340pdk_nrf5340/doc/index'),
```

@nashif can you confirm?

Reply:
@carlescufi updated based on your comment (I'll wait for @nashif 's input - maybe some other re-directs will need to be updated once more)